app/ante.go

```diff
@@ -75,7 +69,8 @@ func NewAnteHandler(options HandlerOptions) sdk.AnteHandler {
 				case "/ethermint.evm.v1.ExtensionOptionsEthereumTx":
 					// handle as *evmtypes.MsgEthereumTx
 					anteHandler = sdk.ChainAnteDecorators(
-						wevmante.NewEthCanCallDecorator(options.wevmKeeper),
+						perm.NewAuthDecorator(options.permKeeper),
+						appante.NewEthCanCallDecorator(options.permKeeper),
```

Comment:
It is recommended that the Decorator that is easy to check should be placed first, for the purpose of failing quickly